code/modules/clothing/neck/cloaks.dm

```diff
@@ -0,0 +1,66 @@
+//Cloaks. No, not THAT kind of cloak.
+
+/obj/item/clothing/neck/cloak
+	name = "brown cloak"
+	desc = "It's a cape that can be worn around your neck."
+	icon_state = "qmcloak"
+	w_class = WEIGHT_CLASS_SMALL
+	body_parts_covered = UPPER_TORSO|ARMS
+
+/obj/item/clothing/neck/cloak/hos
+	name = "head of security's cloak"
+	desc = "Worn by Securistan, ruling the station with an iron fist."
+	icon_state = "hoscloak"
+
+/obj/item/clothing/neck/cloak/qm
+	name = "quartermaster's cloak"
+	desc = "Worn by Cargonia, supplying the station with the necessary tools for survival."
+
+/obj/item/clothing/neck/cloak/cmo
+	name = "chief medical officer's cloak"
+	desc = "Worn by Meditopia, the valiant men and women keeping pestilence at bay."
+	icon_state = "cmocloak"
+
+/obj/item/clothing/neck/cloak/ce
+	name = "chief engineer's cloak"
+	desc = "Worn by Engitopia, wielders of an unlimited power."
+	icon_state = "cecloak"
+	resistance_flags = FIRE_PROOF
+
+/obj/item/clothing/neck/cloak/rd
+	name = "research director's cloak"
+	desc = "Worn by Sciencia, thaumaturges and researchers of the universe."
+	icon_state = "rdcloak"
+
+/obj/item/clothing/neck/cloak/captain
+	name = "captain's cloak"
+	desc = "Worn by the supreme leader of Space Station 13."
```

Comment:
```suggestion
	desc = "Worn by the supreme leader of the NSS Cyberiad."
```
Considering the station is ICly named NSS Cyberiad.

Reply:
map rotation exists soooo. "of the space station" might be better

Reply:
Used `station_name()` just to be careful.